src/util/std_types.h

```diff
@@ -267,77 +248,67 @@ class struct_union_typet:public typet
   void set_tag(const irep_idt &tag) { set(ID_tag, tag); }
 };
 
-/*! \brief Cast a generic typet to a \ref struct_union_typet
- *
- * This is an unchecked conversion. \a type must be known to be \ref
- * struct_union_typet.
- *
- * \param type Source type
- * \return Object of type \ref struct_union_typet
- *
- * \ingroup gr_std_types
-*/
+/// \brief Cast a generic typet to a \ref struct_union_typet
+///
+/// This is an unchecked conversion. \param type must be known to be \ref
+/// struct_union_typet.
+///
+/// \param type Source type
+/// \return Object of type \ref struct_union_typet
 inline const struct_union_typet &to_struct_union_type(const typet &type)
 {
   PRECONDITION(type.id()==ID_struct || type.id()==ID_union);
   return static_cast<const struct_union_typet &>(type);
 }
 
-/*! \copydoc to_struct_union_type(const typet &)
- * \ingroup gr_std_types
-*/
+/// \copydoc to_struct_union_type(const typet &)
 inline struct_union_typet &to_struct_union_type(typet &type)
 {
   PRECONDITION(type.id()==ID_struct || type.id()==ID_union);
   return static_cast<struct_union_typet &>(type);
 }
 
-/*! \brief Structure type
-*/
+/// Structure type
```

Comment:
Not sure this comment adds anything - perhaps just remove or say it corresponds to C style structs